Oswaldo Vazquez Aparicio/26 Agosto/hackerrank/JavaOutputFormatting.java

```diff
@@ -0,0 +1,19 @@
+package hackerrank;
+
+import java.util.Scanner;
+
+public class JavaOutputFormatting {
+
+    public static void main(String[] args) {
+
+            Scanner sc=new Scanner(System.in);
+            System.out.println("================================");
```

Comment:
Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings